src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -336,28 +272,79 @@ class BaseCourseCard extends Component {
     return null;
   };
 
-  renderCourseStartDate = () => {
-    const { startDate, mode } = this.props;
+  renderOrganizationName = () => {
+    const { orgName, mode } = this.props;
+
     const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
+    const course = isExecutiveEducation2UCourse ? 'Executive Education' : 'Course';
+    const tooltipText = isExecutiveEducation2UCourse
+      ? 'Executive Education courses are instructor-led, cohort-based, and follow a set schedule.'
+      : 'Courses are on-demand, self-paced, and include asynchronous online discussion.';
+
+    if (orgName) {
+      return (
+        <p className={`mb-2 font-weight-light ${isExecutiveEducation2UCourse ? 'text-light-300' : ''}`}>
+          {orgName} &#x2022; {course}
+          <OverlayTrigger
+            trigger={['hover', 'focus']}
+            placement="top"
+            overlay={(
+              <Tooltip variant="light" id="tooltip-top">
+                {tooltipText}
+              </Tooltip>
+            )}
+          >
+            <InfoOutline className="ml-2 mb-1" style={{ width: '1rem', height: '1rem' }} />
```

Comment:
nit: would using `Icon` component with `InfoOutline` avoid the need for the custom `style` prop?